src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -6,26 +6,23 @@
         left: auto;
         right: 1rem;
         position: fixed;
-        width: 100%;
         z-index: 1;
 
-        &.jw-float-bottom {
+        @media screen and (min-width: 420px) {
```

Comment:
So there's no default? I don't think you should include this media query because what happens in `landscape` mode at `max-width: 419px`? There should just be small portrait, and everything else.